js/popup.js

```diff
@@ -1,423 +1,396 @@
+    document.getElementById('button_quantimo_google_sign_in').onclick = onQmGoogleButtonClicked;
+    document.getElementById('button_quantimo_sign_in').onclick = onQmSignButtonClicked;
 
-	/*document.getElementById('button-add').onclick=onAddButtonClicked;
-	document.getElementById('button-close').onclick=onCloseButtonClicked;
-	
-	*/
-	
-	document.getElementById('button_quantimo_facebook_sign_in').onclick=onQmFacebookButtonClicked;
-	document.getElementById('button_quantimo_google_sign_in').onclick=onQmGoogleButtonClicked;
-	document.getElementById('button_quantimo_sign_in').onclick=onQmSignButtonClicked;
+    document.getElementById('button-record-a-measurement').onclick = onQmRcdMstButtonClicked;
 
-	document.getElementById('button-record-a-measurement').onclick=onQmRcdMstButtonClicked;
+    document.getElementById('button-edit-record-a-measurement').onclick = onEdtButtonClicked;
+    document.getElementById('button-add-record-a-measurement').onclick = onAddButtonClicked;
+    document.getElementById('anhor-register').onclick = onRegisterAClicked;
 
-	document.getElementById('button-edit-record-a-measurement').onclick = onEdtButtonClicked;
-	document.getElementById('button-add-record-a-measurement').onclick = onAddButtonClicked;
-	document.getElementById('anhor-register').onclick = onRegisterAClicked;
-	
-	document.getElementById('btnQuantiModoRegister').onclick = btnQuantiModoRegisterClick;
+    document.getElementById('btnQuantiModoRegister').onclick = btnQuantiModoRegisterClick;
 
-	document.getElementById('btnQuantiModoRegister_1').onclick = btnQuantiModoRegisterClick;
-	document.getElementById('btnQuantiModoRegister_2').onclick = btnQuantiModoRegisterClick;
+    document.getElementById('btnQuantiModoRegister_1').onclick = btnQuantiModoRegisterClick;
+    document.getElementById('btnQuantiModoRegister_2').onclick = btnQuantiModoRegisterClick;
 
-	document.getElementById('logo-correlate').onclick = btnQuantiModoLogoCorrelateClick;
-	document.getElementById('logo-correlate-1').onclick = btnQuantiModoLogoCorrelateClick;
-	document.getElementById('logo-correlate-2').onclick = btnQuantiModoLogoCorrelateClick;
+    document.getElementById('logo-correlate').onclick = btnQuantiModoLogoCorrelateClick;
+    document.getElementById('logo-correlate-1').onclick = btnQuantiModoLogoCorrelateClick;
+    document.getElementById('logo-correlate-2').onclick = btnQuantiModoLogoCorrelateClick;
 
-	//var optionsUrl = chrome.extension.getURL("src/options/options.html"); 
-	//var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
+    //var optionsUrl = chrome.extension.getURL("src/options/options.html");
+    //var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
 
 
-		
 };
 
 /**
  * Record a measurement is clicked
  */
-var onQmRcdMstButtonClicked = function()
-{
-	var name = $("#addmeasurement-variable-name").val();
+var onQmRcdMstButtonClicked = function () {

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['onQmRcdMstButtonClicked' was used before it was defined.](https://www.codacy.com/app/m_3/QuantiModo-Chrome-Extension/file/842132528/issues/source?bid=2779649&fileBranchId=2805440#l77)